python_modules/dagster/dagster/core/definitions/run_config.py

```diff
@@ -178,12 +180,15 @@ def get_inputs_field(
     dependency_structure: DependencyStructure,
     resource_defs: Dict[str, ResourceDefinition],
     solid_ignored: bool,
+    input_values: Dict[str, Any] = None,
```

Comment:
should probably have a more explicit name given hard to disambiguate in current context 

also `Optional`